tests/test_to_number.cc

```diff
@@ -363,7 +374,7 @@ TEMPLATE_TEST_CASE("to_number(): Convert integers with many leading zeros", "[to
 TEMPLATE_TEST_CASE("to_number(): Convert floating-point values with many leading and trailing zeros", "[to_number]",
     float, double, long double)
 {
-    auto const lenience = 0;
+    auto const lenience = 1;
```

Comment:
Ok, zero lenience makes only sense for leading zero but not after the dot, right?

Reply:
improved